draft-ietf-jose-hpke-encrypt.md

```diff
@@ -541,14 +551,7 @@ for their contributions to the specification.
 
 -13
 
-* Removed orphan text about AKP kty field
-* Fixed bug in "include-fold" syntax
-* Switched reference from RFC 9180 to
-  draft-ietf-hpke-hpke
-* Editorial improvements to abstract and
-  introduction.
-* Removed Section 8.2 "Static Asymmetric
-  Authentication in HPKE"
+* Add HPKE-7
```

Comment:
what is going on here? -13 was published with those things that are deleted. HPKE-7 (and I do think this shows some downsides of the short naming scheme but I digress) would be added in -14